src/auth/ha-auth-flow.ts

```diff
@@ -56,6 +57,21 @@ class HaAuthFlow extends litLocalizeLiteMixin(LitElement) {
         this._handleSubmit(ev);
       }
     });
+
+    window.addEventListener(
```

Comment:
Will this listener remain in effect even when the instance of HaAuthFlow is no longer active?

Reply:
Yes, you would need to remove the listener, or add the option once, so it will be removed when it is invoked.